ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -186,3 +186,29 @@ SENSOR_ZA_PML:
   - zone_air_pm4pt0_volumetricconcentration_sensor
   - zone_air_pm10pt0_volumetricconcentration_sensor
   - zone_air_voc_concentration_sensor
+  
+SENSOR_COC:
+  description: "Carbon monoxide monitoring and control"
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - COC
+  uses:
+  - zone_air_co_concentration_sensor
+ 
+SENSOR_OCO2M:
+  description: "Outside Carbon dioxide monitoring and control"
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - OCO2M
+  uses:
+  - outside_air_co2_concentration_sensor
```

Comment:
you defined this in OCO2M, this isnt needed.

Reply:
removed OCO2M